### ISSUE TITLE
The PC is reachable from the internet using IPv6, nothing blocked (neither inbound nor outbound), no nat, but the "IPv6 enabled" test still fails?!?

### ISSUE THREAD

**Issue by Daniel-Abrecht** [webrtc/testrtc#363]
**Browsers and versions affected**
Chromium "Version 90.0.4430.212 (Developer Build) built on Debian 10.9, running on Debian 10.0 (64-bit)" (This is the official version from the Debian repos)
Firefox "78.11.0esr (64-bit)"

**Steps to reproduce**
Using a PC with IPv6, all ports open, inbound and outbound, udp & tcp, reachable from the Internet, no NAT nor anything else special involved. https://test.webrtc.org/ will show "IPv6 enabled" and "Reflexive connectivity" as not working. 

**Expected results**
"IPv6 enabled" succeeds

**Actual results**
If the PC also has an IPv4 address, "IPv6 enabled" fails (This is for both, chromium and firefox-esr). If the PC doesn't have an IPv4 address (IPv6 only), on chromium, it hangs indefinitely at the "IPv6 enabled" test (the tests before that work). On firefox, all tests fail if IPv6 only.

**Comment**
For IPv6, I verified all ports are reachable from & to the internet, TCP & UDP, inbound and outbound, there is no NAT nor anything like that, and yet, the "IPv6 enabled" and "Reflexive connectivity" check still failed (For both, chromium and firefox).
I even tried without an IPv4 address and with only IPv6, but on chromium, that hung at the "IPv6 enabled" check indefinitely (the other checks before that worked). Firefox with IPv6 only failed completely, but that's another issue.
It seams the issue is not my network, but what else could it be, is it the browser, this test site, or is this broken for everyone in general?

**Comment by Daniel-Abrecht** [comment]
Some people in a forum confirmed for me that it works for them, so I'll have to keep searching for the root cause on my side. I currently suspect some ISP side routing issues, but I'm not sure yet.

**Comment by Daniel-Abrecht** [comment]
I figured out the problem with IPv6 I was having. It turns out, it's the the recently introduces IP anonymization. I've now disabled it in `chromium->about:flags->Anonymize local IPs exposed by WebRTC` and `firefox->about:config->media.peerconnection.ice.obfuscate_host_addresses`.
This also seams to fix some other issues I was having, such as it not finding peers in WebTorrent.
 
 Now I only need to figure out why the "Reflexive connectivity" check doesn't pass yet.
